doc/source/topologies.rst

```diff
@@ -70,7 +70,7 @@ The topology DSL fully supports JVM-based bolts and spouts via the
 :class:`~streamparse.JavaBolt` and :class:`~streamparse.JavaSpout` classes.
 
 Here's an example of how we would use the
-`Storm Kafka Spout <http://storm.apache.org/documentation/storm-kafka.html>`_:
+`Storm Kafka Spout <http://storm.apache.org/releases/1.0.1/storm-kafka.html>`_:
```

Comment:
We should probably make this use `current` instead of `1.0.1`, in case they update those docs with a later release.